portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,22 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+//Causes the bubble to slide down the page.
+function bubbleFall() {
+    //Find the bubble.
+    var elem = document.getElementById('bubble');
+    var pos = 0;
+    //Make the mobement visible and fluid by moving every 10 milliseconds;
+    var id = setInterval(frame, 10)
```

Comment:
nit: missing ;

Reply:
I was wondering if you could elaborate a bit on this comment. Is that a line a am missing in my CSS?

Reply:
Oh my apologies. I see that the 'nit' is just to point something out :)

Reply:
Yeah it's just a nit, not a big deal :)